types/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Plugin } from 'unified'
+import { Node } from 'unist'
+import { Element } from 'hast'
+
+interface RemarkAutolinkHeadingsOptions {
+  behavior?: 'prepend' | 'append' | 'wrap' | 'before' | 'after'
+  linkProperties?: Record<string, any>
+  content?: Element | Element[] | (() => (Element | Element[]))
+  group?: Element | ((node: Node) => Element)
```

Comment:
```suggestion
/**
 * hast node to wrap the heading and link with
 */
  group?: Element | ((node: Node) => Element)
```

(Optional) It could also be a nice to have, to split the interface in two, and have one with `behavior: before` and `behavior: after` with group and one with the rest of the behaviors and no `group`
https://github.com/remarkjs/remark-autolink-headings#optionsgroup